packages/tool-cache/src/tool-cache.ts

```diff
@@ -501,15 +502,16 @@ interface GitHubTree {
 export async function getManifestFromRepo(
   owner: string,
   repo: string,
-  token: string,
+  auth?: string,
```

Comment:
Technically could have left getManifestFromRepo as is... Ended up changing for consistency within the module (downloadTool). Also consistency within the scenario (call getManifestFromRepo then downloadTool).

lmk and i can revert